pages/home/registry-early-access.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Arcade Registry Early Access"
+description: "Learn about the Arcade Registry and how to get early access"
+---
+
+import { EarlyAccessRegistrySurvey } from "@/components/EarlyAccessRegistrySurvey";
+
+# The Arcade Registry
+
+The **Arcade Registry** is how agentic tool developers share their work with the world.  Arcade.dev is collecting all the integrations that agents will ever need in one place -  think HuggingFace or Pypi but for LLM tools.  A powerful community of open source and for-profit developers building out robust and evaluated workflows is how elevate agents to being *truly* useful.
+
+Unlike traditional read-only tool libraries, Arcade.dev couples the runtime with the registry.  This way we can collect real metrics and usage information about your tools, sharing meaningful information and feedback back to the developers.  You'll get error reports and statistics about how your tools are getting used.  Arcade.dev also makes it possible for developers to optionally choose to sell their tools with a markup on top of the Arcade platform fees.
+
+Thanks to the Arcade Gateway, your toolkits will be available via all the protocols Arcade supports - be that MCP for locally-running applications, OXP for server applications, and whatever comes next.  Use the open-source Arcade SDKs to future-proof your tools.
+
+We are seeking beta testers who are interested in building, maintaining, and sharing toolkits with the world in either an free-to-use or for-profit manner.  Sign up today to join the Arcade Registry Beta.
```

Comment:
`either an free-to-use` --> `either a free-to-use`

Reply:
fixed up in 6f5f11151c036fb60e4e834130acacebb9b6f816